### DISCUSSION TITLE
How do you refresh a member list?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#794] [IBM i] [answered]
I check out source to my developers library. If the list of members is open example QDDSSRC, I cannot find a way to refresh the list to see new members I have checked out. 

How do we do this?

TIA, Darryl Freinkel.

**Comment by worksofliam** [comment, marked as answer]
There is a refresh button (and refresh icon) at the top of the browser view. The icons are visible when your mouse is over the Object Browser
